src/schema/rules/files/raw/ephys.yaml

```diff
@@ -0,0 +1,19 @@
+---
+ephys:
+  suffixes:
+    - ephys
+  extensions:
+    - .nwb
+    # possible future: serialization in .zarr format to accompany .ome.zarr
+    # - .nwb.zarr
+    - .nix
+  datatypes:
+    - ephys
+  entities:
+    subject: required
+    session: optional
+    sample: optional
+    task: optional
+    acquisition: optional
+    run: optional
+    split: optional
```

Comment:
currently we have a test failing
```
❯ python -m pytest -s -v --tb=short bidsschematools/data/tests/test_rules.py::test_rule_objects
======================================== test session starts ========================================
platform linux -- Python 3.11.8, pytest-8.1.1, pluggy-1.4.0 -- /home/yoh/proj/bids/bids-specification/tools/schemacode/venvs/dev3/bin/python
cachedir: .pytest_cache
rootdir: /home/yoh/proj/bids/bids-specification/tools/schemacode
configfile: pyproject.toml
plugins: cov-4.1.0
collected 1 item                                                                                    

bidsschematools/data/tests/test_rules.py::test_rule_objects FAILED

============================================= FAILURES ==============================================
_________________________________________ test_rule_objects _________________________________________
bidsschematools/data/tests/test_rules.py:103: in test_rule_objects
    raise ValueError(not_found_string)
E   ValueError: files.raw.channels.channels__ephys.datatypes[0] == ephys
E   files.raw.channels.coordsystem__ephys.datatypes[0] == ephys
E   files.raw.channels.coordsystem__ephys.datatypes[1] == iephys
E   files.raw.ephys.ephys.datatypes[0] == ephys
---------------------------------------- Captured log setup -----------------------------------------
INFO     bids-schema:schema.py:193 No schema path specified, defaulting to the bundled schema, `/home/yoh/proj/bids/bids-specification/tools/schemacode/bidsschematools/data/schema`.
====================================== short test summary info ======================================
FAILED bidsschematools/data/tests/test_rules.py::test_rule_objects - ValueError: files.raw.channels.channels__ephys.datatypes[0] == ephys
```
which might explain @TheChymera why validator also pukes (you mentioned that in the matrix chat).  Could you have a look/address for this test fail?

Reply:
@yarikoptic yes, this is the issue I ran into, I should have pasted it in the chat. I'm not sure what the problem is — I define the datatype here. Maybe this also needs to be done in some other file 🤔

Reply:
we have troubleshooted/fixed this one